paddlets/analysis/time_domain.py

```diff
@@ -364,8 +364,8 @@ class Correlation(Analyzer):
 
     """
     def __init__(self, 
-                 method: str = 'pearson',
-                 lag: int = 0,
+                 method: Optional[str] = 'pearson',
+                 lag: Optional[int] = 0,
```

Comment:
should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional